client/StatusTable/components/scss/StatusTable.module.scss

```diff
@@ -0,0 +1,53 @@
+.status-table {
+  border: {
+    spacing: 0;
+  }
+  text-align: center;
+}
+
+.status-table {
+  &__col-day {
+    width: 260px;
+  }
+  &__col-number {
+    width: 50px;
+  }
+  &__col-start {
+    width: 150px;
+  }
+  &__col-finish {
+    width: 145px;
+  }
+  &__col-time {
+    width: 140px;
+  }
+  &__col-real-time {
+    width: 120px;
+  }
+}
+
+.status-table__head-cell {
+  font: {
+    family: 'Roboto', sans-serif;
+    size: 0.85em;
+    weight: normal;
+  }
+  padding: 5px;
+  height: 40px;
+  background: white;
+  position: sticky;
+  position: -webkit-sticky;
+  top: 0;
+  border-bottom: 1px solid #CACACA;
```

Comment:
I would keep some colors as a variable in some shared place so you can easily change color scheme of the project